examples/ref-templates/scripts/htlc.js

```diff
@@ -0,0 +1,40 @@
+const { transferMicroAlgos, transferMicroAlgosLsig } = require("algob");
+
+async function transferAlgo(deployer, senderAddr, receiverAddr, amount, lsig, payFlags){
+    try {
+        const details = await transferMicroAlgosLsig(deployer, senderAddr, receiverAddr, amount, lsig, payFlags);
+        console.log(details);
+    } catch (e) {
+        console.error('Transaction Failed', e);
+    }
+}
+
+async function run(runtimeEnv, deployer) {
+
+  const masterAccount = deployer.accountsByName.get("master-account")
+  const johnAccount = deployer.accountsByName.get("john-account");
+  const globalZeroAddress = "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAY5HFKQ";
```

Comment:
Le's move this to algob to `src/lib/constants.ts` and import it here.

Reply:
ok